migrate.py

```diff
@@ -23,9 +22,16 @@
 from logzero import logger
 
 from baron.parser import ParsingError
+from ruamel.yaml import YAML
```

Comment:
i'm trying to avoid parsing corner cases by using the same yaml lib as core, I don't care much about retaining formatting but if you really want to i would prefer to do a replace on the source than reconstruct the yaml.